packages/flet_video/lib/src/video.dart

```diff
@@ -68,6 +68,12 @@ class _VideoControlState extends State<VideoControl> with FletStoreMixin {
       .triggerControlEvent(widget.control.id, "completed", message ?? "");
   }
 
+  void _onTrackChanged(String? message) {
+  debugPrint("Video onTrackChanged: $message");
+  widget.backend
+      .triggerControlEvent(widget.control.id, "track", message ?? "");
```

Comment:
I would use `"track_changed"` for the event string. Change on Python side as well.